_data/community-stories.json

```diff
@@ -1306,5 +1306,283 @@
     "user_agent": "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/90.0.4430.212 Safari/537.36",
     "referrer": "https://www.crossroads.net/my25/",
     "created_at": "2021-05-26T14:51:16.882Z"
-  }
-]
\ No newline at end of file
+  },
+  [
```

Comment:
Please Remove this `[`.

The overall structure should be.

```json
[
  {},
  {},
  {},
  {}
]
```

what you have is.
```json
[
  {},
  {},
  [
      {},
      {}
  ]
]
```